### PR TITLE
Replaced obsolete six@gpu with six@v100

### PR DESCRIPTION
**Pull request by TevenLeScao**
Due to JZ changes, the `@gpu` flag in account choice will be phased out in favour of `@v100`. This PR updates the flag in the training scripts. I've checked once that nothing was broken, but if someone else wants to double check please do!

### CODE REVIEW

**Review by stas00 — APPROVED** [review]
Thank you, Teven!

File: train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm [hunk 1]
```diff
@@ -104,7 +104,7 @@ GPT_ARGS=" \
     --seq-length $SEQ_LEN \
     --max-position-embeddings $SEQ_LEN \
     --micro-batch-size $MICRO_BATCH_SIZE \
-    --rampup-batch-size 192 16 9_765_625 \
+    --rampup-batch-size 192 32 9_765_625 \
```
Comment (SaulLu) [inline train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm:107]: I'm just taking the opportunity to raise a change here that doesn't seem to me to be related to change `@gpu` to `@v100` (especially if it's not a desirable change)

**Review by SaulLu — APPROVED** [review]
LGTM thanks!

File: train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm (hunk 1 above)
Comment (TevenLeScao) [reply train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm:107]: ah it's a desirable change, the 16-increment breaks a multiple-of-32 assumption and it should indeed be 32. I should have merged this earlier though. I'm a bit lazy to make it a separate PR but if someone thinks it's important I'll separate it

Comment (TevenLeScao) [reply train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm:107]: thanks for spotting it!

Comment (SaulLu) [reply train/tr11-176B-ml/smaller_models/tr11e-350M-ml.slurm:107]: Thanks for the explanation :smile: 
Fine with me to keep it here!

**Comment by TevenLeScao** [comment]
OK, merging, thanks for the check guys

**Pull request merged**